http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfigurationBuilder.java

```diff
@@ -19,6 +19,8 @@
 package org.wildfly.security.http.oidc;
 
 import static org.wildfly.security.http.oidc.ElytronMessages.log;
+import static org.jose4j.jws.AlgorithmIdentifiers.NONE;
```

Comment:
Very minor, this import should come before the org.wildfly one (since we use alphabetical order).